trunk/SUAVE/Methods/Aerodynamics/AVL/create_avl_datastructure.py

```diff
@@ -82,7 +82,7 @@ def translate_avl_geometry(geometry):
                 aircraft.append_wing(w)
                 
         for body in geometry.fuselages:
-                if body.tag == 'fuselage':
+                if body.configuration == 'tube_and_wing' or body.configuration == 'boom':
```

Comment:
see above comment about configurations

Reply:
I corrected the other file so issues is now resolved